packages/VersionBonding/Application/MissedRectorDueVersionChecker.php

```diff
@@ -84,20 +84,25 @@ private function reportMissedRectors(array $minPhpVersions): void
     }
 
     /**
-     * @param MinPhpVersionInterface[] $minPhpVersions
+     * @param MinPhpVersionInterface[] $missedRectors
      */
-    private function reportWarningMessage(int $minProjectPhpVersion, array $minPhpVersions): void
+    private function reportWarningMessage(int $minProjectPhpVersion, array $missedRectors): void
     {
         $phpVersion = new PhpVersion($minProjectPhpVersion);
 
         $warningMessage = sprintf(
-            'Your project requires min PHP version "%s".%s%d%sSome Rectors rules defined in your configuration require higher PHP version and will not run,%sto avoid breaking your codebase.',
+            'Your project requires min PHP version "%s".%s%sThe following Rector rules defined in your configuration require higher PHP version and will not run,%sto avoid breaking your codebase: ',
             $phpVersion->getVersionString(),
-            count($minPhpVersions),
             PHP_EOL,
             PHP_EOL . PHP_EOL,
             PHP_EOL
         );
+
+        $warningMessage .= PHP_EOL;
+        foreach ($missedRectors as $missedRector) {
+            $warningMessage .= PHP_EOL . ' - ' . $missedRector::class;
+        }
+
```

Comment:
This should be only displayed on `-vvv`, so we don't bother user with detailed info.

Reply:
Ok, updated with `-vvv` hint if user want to see the detailed rector rules 

![Screen Shot 2021-08-02 at 21 23 38](https://user-images.githubusercontent.com/459648/127876848-d1a69277-4c27-4c76-a8a4-ee17ea8ec78a.png)

Reply:
I think it is ready with correctly show counted rector rules -> previously shows 140, which should be 14

![Screen Shot 2021-08-02 at 21 30 47](https://user-images.githubusercontent.com/459648/127877934-0b4f13f4-2f89-4ead-b8b7-4117e12debd8.png)